### PR TITLE
Chef 13 fixes for osl-keepalived

### PR DESCRIPTION
**Pull request by TravisWhitehead**
It seems that this one didn't make it onto our trello card.

### CODE REVIEW

**Comment by TravisWhitehead** [comment]
[Chef 13: osl-keepalived](https://trello.com/c/NAojPbYT/1070-chef-13-osl-keepalived)

File: test/integration/provisioning/keepalived.rb [hunk 1]
```diff
@@ -6,10 +6,10 @@
 [%w(node1 11), %w(node2 12)].each do |name, ip_suff|
   machine name do
     machine_options vagrant_options: {
-      'vm.box' => 'bento/centos-7.4'
+      'vm.box' => 'bento/centos-7.4',
     },
                     convergence_options: {
-                      chef_version: '12.18.31'
+                      chef_version: '12.18.31',
```
Comment (ramereth) [inline test/integration/provisioning/keepalived.rb]: Please bump this up to 13.8.5

File: test/integration/provisioning/keepalived.rb [hunk 2]
```diff
@@ -6,10 +6,10 @@
 [%w(node1 11), %w(node2 12)].each do |name, ip_suff|
   machine name do
     machine_options vagrant_options: {
-      'vm.box' => 'bento/centos-7.4'
+      'vm.box' => 'bento/centos-7.4',
```
Comment (ramereth) [inline test/integration/provisioning/keepalived.rb]: Please change this to ``bento/centos-7``

**Comment by TravisWhitehead** [comment]
@ramereth Done!

**Comment by TravisWhitehead** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.2.4.  Have a nice day!

**Pull request merged**